index.html

```diff
@@ -162,28 +191,34 @@ <h4>HTML</h4>
                             <h4>CSS</h4>
                         </li>
                     </ul>
-        
+
                     <ul class="tools-container">
                         <li class="tools">framworks <span class="more">></span></li>
                         <li class="tools">skills <span class="more">></span></li>
                     </ul>
                 </article>
```

Comment:
- Kindly make sure technologies, frameworks, and skills are on the same list`<ul>` and they respect the Figma design in terms of margins,paddings and color.


- Your design:
![Screenshot from 2023-03-30 11-22-15](https://user-images.githubusercontent.com/97834160/228791768-2ceb3500-9fba-427a-a753-f52f3271305c.png)

- Figma:
![Screenshot from 2023-03-30 11-25-11](https://user-images.githubusercontent.com/97834160/228791902-044715ca-a1d7-4d7e-accc-7c31cb46b4eb.png)